### ISSUE TITLE
Tags

### ISSUE THREAD

**Issue by tj** [apex/apex#27]
Can't remember if Lambda functions support tags but that'd be helpful for billing etc.

**Comment by mthenw** [comment]
Nope, no tags yet.

**Comment by davyzhang** [comment]
![image](https://cloud.githubusercontent.com/assets/576863/25266881/698bf47a-26a5-11e7-8264-a84b36e353cf.png)
looks like they have tag support now

**Comment by maschek** [comment]
Yes, they support now. See: https://aws.amazon.com/about-aws/whats-new/2017/04/aws-lambda-supports-tagging-and-cost-allocations/
Would be nice to have it in apex project and function json.

**Comment by bonclay7** [comment]
Hi, I would like to contribute on this feature as my first contribution on apex. Is it possible ?
